tox.ini

```diff
@@ -70,67 +70,72 @@ envlist =
+    pymemcache_contrib{,_autopatch}-{py27,py34,py35,py36,py37,py38}-pymemcache{130,140}
+    pymongo_contrib-{py27,py34,py35,py36,py37,py38}-pymongo{30,31,32,33,34,35,36,37,38,39,latest}-mongoengine{latest}
+    pymysql_contrib-{py27,py34,py35,py36,py37,py38}-pymysql{07,08,09}
+    pyramid_contrib{,_autopatch}-{py27,py34,py35,py36,py37,py38}-pyramid{17,18,19}-webtest
+    redis_contrib-{py27,py34,py35,py36,py37,py38}-redis{26,27,28,29,210,300}
+    rediscluster_contrib-{py27,py34,py35,py36,py37,py38}-rediscluster{135,136}-redis210
+    requests_contrib{,_autopatch}-{py27,py34,py35,py36,py37,py38}-requests{208,209,210,211,212,213,219}
     kombu_contrib-{py27,py34,py35,py36}-kombu{40,41,42}
     # Python 3.7 needs Kombu >= 4.2
-    kombu_contrib-py37-kombu42
+    kombu_contrib-py{37,38}-kombu42
 # python 3.6 requests + gevent regression test
 # DEV: This is a known issue for gevent 1.1, suggestion is to upgrade to gevent > 1.2
 #      https://github.com/gevent/gevent/issues/903
     requests_gevent_contrib-{py36}-requests{208,209,210,211,212,213,219}-gevent{12,13}
-    requests_gevent_contrib-py37-requests{208,209,210,211,212,213,219}-gevent13
-    sqlalchemy_contrib-{py27,py34,py35,py36,py37}-sqlalchemy{10,11,12}-psycopg228-mysqlconnector
-    sqlite3_contrib-{py27,py34,py35,py36,py37}-sqlite3
-    tornado_contrib-{py27,py34,py35,py36,py37}-tornado{40,41,42,43,44,45}
-    tornado_contrib-{py37}-tornado{50,51,60}
+    requests_gevent_contrib-py{37,38}-requests{208,209,210,211,212,213,219}-gevent13
+    sqlalchemy_contrib-{py27,py34,py35,py36,py37,py38}-sqlalchemy{10,11,12}-psycopg228-mysqlconnector
+    sqlite3_contrib-{py27,py34,py35,py36,py37,py38}-sqlite3
+    tornado_contrib-{py27,py34,py35,py36,py37,py38}-tornado{40,41,42,43,44,45}
+    tornado_contrib-py{37,38}-tornado{50,51,60}
     tornado_contrib-{py27}-tornado{40,41,42,43,44,45}-futures{30,31,32}
-    vertica_contrib-{py27,py34,py35,py36,py37}-vertica{060,070}
+    vertica_contrib-{py27,py34,py35,py36,py37,py38}-vertica{060,070}
 # Opentracer
-    {py27,py34,py35,py36,py37}-opentracer
-    {py34,py35,py36,py37}-opentracer_asyncio
-    {py34,py35,py36,py37}-opentracer_tornado-tornado{40,41,42,43,44}
+    {py27,py34,py35,py36,py37,py38}-opentracer
+    {py34,py35,py36,py37,py38}-opentracer_asyncio
+    {py34,py35,py36,py37,py38}-opentracer_tornado-tornado{40,41,42,43,44}
     {py27}-opentracer_gevent-gevent{10}
     {py27,py34,py35,py36}-opentracer_gevent-gevent{11,12}
-    py37-opentracer_gevent-gevent{13,14}
+    py{37,38}-opentracer_gevent-gevent{13,14}
 # Unit tests: pytest based test suite that do not require any additional dependency
-    unit_tests-{py27,py34,py35,py36,py37}
-    benchmarks-{py27,py34,py35,py36,py37}
+    unit_tests-{py27,py34,py35,py36,py37,py38}
+    benchmarks-{py27,py34,py35,py36,py37,py38}
 
 [testenv]
+# Newer versions of Tox have issues decoding unicode output in Python 2.7
+# https://github.com/tox-dev/tox/issues/1369
+setenv =
+    LC_ALL=en_US.UTF-8
```

Comment:
that seems unrelated to the PR and not something I encountered anywhere yet?
it'd be good to have this in a different PR since it's a different topic.